Makefile

```diff
@@ -84,11 +85,15 @@ install: $(wildcard *.go)
 	go install -v -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)'
 
 .PHONY: build
-build: $(EXECUTABLE)
+build: $(EXECUTABLE) templates/.VERSION
 
 $(EXECUTABLE): $(wildcard *.go)
 	go build -v -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
 
+.PHONY: templates/.VERSION
+templates/.VERSION:
+	echo -n $(VERSION) > $@
```

Comment:
the HELL apple, stop fucking around with my flags! 😆 

I'll trim it instead... maybe I should drop it? I think that @tboerger PR for removing it is merged no?